#157731607 Format code to PEP8 & Flake8 python style standard

hariclerry opened the pull request:
### What does this PR do?
Format code to meet PEP8 python coding style.

#### Description of Task to be completed?
1. Install flake8 and fix all pep8 issues.

#### How should this be manually tested?
1. Run the commands below
```
$ git clone https://github.com/andela/wg-rogue-one.git
$ cd wg-rogue-one

$ git checkout ch-implement-python-coding-style-157731607

```
2. follow the setup instructions in [README.rst](https://github.com/andela/wg-rogue-one/blob/ch-implement-python-coding-style-157731607/README.rst)
3. run the command ```pip install requirements.txt file```
4. run the command ```autopep8 -i -r --global-config setup.cfg wg-rogue-one```
5. run the command ```flake8 --config setup.cfg  wg-rogue-one```

#### Any background context you want to provide?
To check for Pep8 issues, run the above commands with specific file name or folder name.
  
#### What are the relevant pivotal tracker stories?
#157731607

hariclerry commented:
Hey guys, can you please review my code and let me know what you think. Am still fixing a few issues though.

tibetegya commented on .gitignore:
```diff
@@ -21,6 +21,8 @@ var/
 *.egg-info/
 .installed.cfg
 *.egg
+.DS_Store
```
put a * before .DS_Store since some instances of that file are nested in other folders

dondrzzy replied:
Plus 1

tibetegya commented on requirements.txt:
```diff
@@ -1,30 +1,50 @@
-
-# Python3 compatibility
-six
+alabaster==0.7.10
```
I recommend you only add the packages you manually installed, their dependancies will automatically be installed, you do not have to add them here.

hariclerry replied:
I will work on that.

tibetegya commented on setup.cfg:
```diff
@@ -5,3 +5,8 @@ universal = 1
+[flake8]
+exclude = .git,__pycache__,urls.py,tasks.py,*migrations*,*bower_components*,*docs*,__*__,
+max-line-length = 100
+ignore = F401,W503
```
add a new line at the end of your files

hariclerry replied:
@tibetegya, there is a new line but I don't know why it is not being shown here.

Commit:
feat(All-exercise-info-feature) Creates an exercise info read-only API endpoint

This commit creates a new read-only API endpoint that collects all the information
of an exercise such as muscles, category, images, etc.

Add a new exercise-readonly API endpoint to wger/urls.py,
Add an ExerciseReadOnlyViewSet class to wger/exercises/api/views.py and initialize
it as a ReadOnlyModelViewSet to enable only the GET parameter,
Add ExerciseReadOnlySerializer class to wger/exercises/api/serializers.py to convert
the data into JSON format.

[Delivers ##157731611]

dondrzzy reviewed (commented):
Some good work Harriet. This shows that pep8 is doing its work.

emugaya commented:
@hariclerry push your code to GitHub again for the build to pass

hariclerry commented:
@dondrzzy, great. @emugaya, I have pushed to Github again and hope the build passes.

emugaya reviewed (changes requested):
Resolve merge conflicts

hariclerry commented:
@emugaya, merge conflicts resolved.

emugaya commented:
@hariclerry I have failed to test your PR using the instructions you give on the PR. Which folder should be in to run these commands?

hariclerry commented:
@emugaya, you first run the commands on the wger folder first e.g, 
autopep8 -i -r --global-config setup.cfg wger
flake8 --config setup.cfg wger

but the commands should work on the entire project folder.

hariclerry commented:
@emugaya, I have also updated the PR.

emugaya commented:
@hariclerry What is the output I should get after running the above commands? `autopep8 -i -r --global-config setup.cfg wger` does not give any output.

hariclerry commented:
@emugaya, when you run the commands and do not see any output, it means there are no issues/error with the pep8 standard. It means all is well. For testing purpose, open the setup.cfg file and remove docs folder under the flake8 config settings and run the command for flake8 again, you should be able to see some error messages displayed as output in the terminal.
